include/wx/pen.h

```diff
@@ -59,6 +59,86 @@ enum wxPenCap
     wxCAP_BUTT
 };
 
+// ----------------------------------------------------------------------------
+// wxPenInfo describes a wxPen
+// ----------------------------------------------------------------------------
+
+class wxPenInfo
+{
+public:
+    wxPenInfo() :
+        m_colour(wxNullColour)
+    {
+        Init();
+
+        m_width = 1;
+        m_style = wxPENSTYLE_SOLID;
+    }
+
+    explicit wxPenInfo(const wxColour& colour, int width = 1, wxPenStyle style = wxPENSTYLE_SOLID) :
```

Comment:
In principle, by adding `= wxColour()` to the first argument, this ctor could replace the default ctor too. Notice that, unlike in `wxPen` case, there is no advantage in having a separate implementation for the default ctor (for `wxPen` there is, as the default ctor doesn't create any pen object, while the other one does, but this is irrelevant here).